frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalette.tsx

```diff
@@ -81,27 +93,22 @@ export const CommandPalette: FC = () => {
           <Command.Empty>No results found.</Command.Empty>
           <Command.Group heading="Tables">
             {Object.values(schema.current.tables).map((table) => (
-              <Command.Item
-                key={table.name}
-                value={table.name}
-                onSelect={() => goToERD(table.name)}
-                data-focused={focusedTableName === table.name}
-              >
-                {/** biome-ignore lint/a11y/useKeyWithClickEvents: Keyboard interaction is implemented in the parent Command.Item component's onSelect handler. */}
-                <div
-                  className={styles.itemInner}
+              <Command.Item key={table.name} value={table.name} asChild>
+                <a
+                  href={`?active=${table.name}`}
                   onClick={(event) => {
-                    event.stopPropagation()
-                    setTableName(table.name)
-                    setFocusedTableName((prev) =>
-                      prev === table.name ? null : table.name,
-                    )
+                    // Do not call preventDefault to allow the default link behavior when ⌘ key is pressed
+                    if (event.ctrlKey || event.metaKey) {
+                      return
+                    }
```

Comment:
For mouse use, it behaves as a normal link element while ⌘ (or ctrl) being hold.